core/survey/_survey/surveyNodeDefsIndex.js

```diff
@@ -8,19 +8,19 @@ const getNodeDefSource = (nodeDef) => (survey) =>
   NodeDef.isVirtual(nodeDef) ? getNodeDefByUuid(NodeDef.getUuid)(survey) : null
 
 const calculateNodeDefChildren =
-  (nodeDef, includeAnalysis = false) =>
+  (nodeDef) =>
   (survey) => {
     const children = []
     if (NodeDef.isVirtual(nodeDef)) {
       // If nodeDef is virtual, get children from its source
       const entitySource = getNodeDefSource(nodeDef)(survey)
-      children.push(...calculateNodeDefChildren(entitySource, includeAnalysis)(survey))
+      children.push(...calculateNodeDefChildren(entitySource)(survey))
     }
 
     const nodeDefUuid = NodeDef.getUuid(nodeDef)
     children.push(
       ...Object.values(survey.nodeDefs).filter((nodeDefCurrent) => {
-        if (!includeAnalysis && NodeDef.isAnalysis(nodeDefCurrent)) {
+        if (NodeDef.isAnalysis(nodeDefCurrent)) {
```

Comment:
in this case the analysis node defs will always be excluded. will they show up in the data explorer for example?